bundles/org.eclipse.swt/Eclipse SWT Custom Widgets/common/org/eclipse/swt/custom/CTabFolderRenderer.java

```diff
@@ -98,6 +98,7 @@ public class CTabFolderRenderer {
 	static final int ITEM_LEFT_MARGIN = 4;
 	static final int ITEM_RIGHT_MARGIN = 4;
 	static final int INTERNAL_SPACING = 4;
+	static final int HIGHLIGHT_BAR_THICKNESS = Boolean.getBoolean("SWT_CTABFOLDER_DISABLE_HIGHLIGHT") ? 0 : 2;
```

Comment:
The convention for system flags differs from environment variables, so `org.eclipse.swt.CTabFolder.disableHighlight` would be the more consistent key.